### ISSUE TITLE
Can any user with dotcom account access attribution?

### ISSUE THREAD

**Issue by cbart** [sourcegraph/sourcegraph#59777]

**Comment by cbart** [comment]
Yes.

<img width="1037" alt="Screenshot 2024-01-25 at 10 54 50 PM" src="https://github.com/sourcegraph/sourcegraph/assets/153410/d7d6b13f-3a12-42ec-b6db-07ca833eac4c">

**Comment by cbart** [comment]
Let's harden this.

**Comment by cbart** [comment]
@andreeleuterio the graphql op is technically accessible by anyone with dotcom account, but it just runs search, so it does not technically open any extra capacity. Via gateway only product subscriptions can access the endpoint. Does it make sense to harden this:

- before release?
- at all?

(the good thing is that this is on dotcom, so we technically do not need to sync with the release date)

**Comment by andreeleuterio** [comment]
@cbart I agree it doesn't have big impact currently since it only repurposes existing functionality. But as Guardrails expands I can imagine this query being more and more expensive. It's not a release-blocking issue. We can follow-up after the release. Thanks a lot for confirming!
